org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/managers/StandardProjectsManager.java

```diff
@@ -402,6 +404,11 @@ public void preferencesChange(Preferences oldPreferences, Preferences newPrefere
 						registerWatcherJob.schedule(1000L);
 						UpdateClasspathJob.getInstance().updateClasspath();
 					}
+					if ((!Objects.equals(oldPreferences.getSha256Allowed(), newPreferences.getSha256Allowed()) || !Objects.equals(oldPreferences.getSha256Disallowed(), newPreferences.getSha256Disallowed()))) {
```

Comment:
this belongs to a Gradle specific Preference Listener

Reply:
Fixed.